src/commands/stac-validate/stac.validate.ts

```diff
@@ -55,7 +55,14 @@ export const commandStacValidate = command({
     const validated = new Set<string>();
 
     const recursive = args.recursive;
-    const paths = args.location.map((c) => c.trim());
+
+    if (args.location[0] === undefined) {
+      logger.error('StacValidation:Error:NoLocationProvided');
+      process.exit(1);
+    }
+    console.log(args.location);
```

Comment:
Are they "debug" logging?

Reply:
Ha I forgot I just added that to test the input.